joomla/joomla-cms
# [3.x] Add joomla-backports/phpmailer to CMS repo

### Summary of Changes
Replaced the "original" PHPMailer library with the version from https://github.com/joomla-backports/PHPMailer


### Testing Instructions
1. Set up site with working with working mail system (SMTP, mail(), sendmail). 

2. Send test mail using the according button in the global configuration, verify mail arrived

3. apply patch

4. repeat step 2


### Actual result BEFORE applying this Pull Request
Mail is sent.


### Expected result AFTER applying this Pull Request
Mail is sent.


### Documentation Changes Required
None
